src/common/log.js

```diff
@@ -2,6 +2,7 @@
 
 const { join } = require('path')
 const { assign } = Object
```

Comment:
I tried to make use of `...` but could not make it work, as the module would export an object, not a function.